### PR TITLE
Add packetized telemetry option

### PR DESCRIPTION
**Pull request by timcanham**
| | |
|:---|:---|
|**_Originating Project/Creator_**| Ingenuity |
|**_Affected Component_**| TlmPacketizer  |
|**_Affected Architectures(s)_**| all |
|**_Related Issue(s)_**|  |
|**_Has Unit Tests (y/n)_**| y |
|**_Builds Without Errors (y/n)_**| y |
|**_Unit Tests Pass (y/n)_**| y |
|**_Documentation Included (y/n)_**| y |

---
## Change Description

Add packetized telemetry component and cmake autocoder to F Prime

## Rationale

This allows users to use fixed packets instead of streaming telemetry for telemetry channels.

## Testing/Review Recommendations

Cmake and autocoder changes can be reviewed.

## Future Work

None

### CODE REVIEW

File: Svc/TlmPacketizer/test/ut/main.cpp [hunk 1]
```diff
@@ -3,64 +3,55 @@
...
-
-    TEST_CASE(100.1.2,"Send Packets");
+TEST(TestNominal, SendPackets) {
+    TEST_CASE(100.1.2, "Send Packets");
     Svc::Tester tester;
     tester.sendPacketsTest();
 }
 
-//TEST(TestNominal,SendPacketLevels) {
+// TEST(TestNominal,SendPacketLevels) {
```
Comment (LeStarch) [inline Svc/TlmPacketizer/test/ut/main.cpp:28]: Is this not needed?  If so, remove?

**Review by LeStarch — APPROVED** [review]
Few minor comments.  Most of these comments apply to what is already in `devel`.

**Review by thomas-bc — APPROVED** [review]
Reviewed the CMake part only, looks good to me!

**Comment by LeStarch** [comment]
The outstanding comments were already passed in the original PR.  This is the script/delta. Merging.

**Pull request merged**